download/downloadSpec.go

```diff
@@ -0,0 +1,221 @@
+package download
+
+import (
+	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+
```

Comment:
is this library still supported ?

Reply:
agreed to raise card to review library

Reply:
See https://github.com/ONSdigital/dp-mongodb-in-memory/pull/6